system-test/Dockerfile.node10-alpine

```diff
@@ -7,6 +7,7 @@ RUN go get github.com/google/pprof
 FROM node:10-alpine
 
 ARG ADDITIONAL_PACKAGES
+ENV ALPINE=true
```

Comment:
It's unnecessary complexity in my opinion to stream down a special value just to recognize the alpine environment.

Is there a simpler local check you could make? Such as check existence of /bin/busybox file? Like below but I didn't test it.

function timeout() {
  if [ -f /bin/busybox ]; then
    timeout -t "${@}"
  else
    timeout "${@}"
  fi
}

Reply:
Thanks! That's much simpler!

Locally using "timeout" as the function name caused "timeout" to call itself. So, I called it "wait_only".